pyk/src/pyk/ktool/kprove.py

```diff
@@ -143,7 +143,7 @@ def prove(
             proc_output = run_process(command, _LOGGER, env=command_env).stdout
         except CalledProcessError as err:
             if err.returncode != 1:
-                raise RuntimeError(f'Command kprove exited with code {err.returncode} for: {spec_file}', err.stdout, err.stderr)
+                raise RuntimeError(f'Command kprove exited with code {err.returncode} for: {spec_file}', err.stdout, err.stderr) from err
```

Comment:
Note that exceptions will be implicitly chained here even if `from` is not provided. (I'm fine with making this explicit though.)

Reply:
Oh, I've been seeing very different error messages since I've started chaining explicitly. The previous messages made me mildly panic everytime I saw them thinking something went wrong when trying to handle the exception.